### PR TITLE
#1586 - Bug - DataCleaner/DataCloud status panel/"Sign in to DataClou…

### LINKED ISSUE #1586: DataCleaner/DataCloud status panel/"Sign in to DataCloud" button does nothing
Description
When I open "fresh" DataCleaner and check "do not show again" checkbox on "Sign in to DataCloud welcome panel" and then open "DataCloud status panel" by clicking on the bottom "DataCloud" icon, the "Sign in to DataCloud" button is available and clickable but does nothing (at least from user's point of view). 

Expected Behavior
a) Option "do not show again" is related only to "welcome dialog" and not to the "Sign in to DataCloud" button. 
b) When I previously selected "do not show again", this "Sign in to DataCloud" button should not be available/visible. 

Observed Behavior
"Sign in to DataCloud" button only closes the "DataCloud status" panel but does not show any login form or a web page in a browser.

### PR DESCRIPTION
**Pull request by khouzvicka**
fixes #1586

### CODE REVIEW

File: desktop/ui/src/main/java/org/datacleaner/windows/DataCloudLogInWindow.java [hunk 1]
```diff
@@ -309,9 +315,12 @@ private DCPanel contentSuccessCard() {
     }
 
     private DCPanel contentBottom() {
+        String showDataCloudDialog = _userPreferences.getAdditionalProperties()
+                .getOrDefault(SHOW_DATACLOUD_DIALOG_USER_PREFERENCE, "false");
+        Boolean showDataCloudDialogBool = Boolean.parseBoolean(showDataCloudDialog);
 
         final JButton closeButton = WidgetFactory.createDefaultButton("Close", IconUtils.ACTION_CLOSE_DARK);
-        dontShowAgainCheckBox = new JCheckBox("Don't show again.", false);
+        dontShowAgainCheckBox = new JCheckBox("Don't show after start again.", !showDataCloudDialogBool);
```
Comment (ClaudiaPHI) [inline desktop/ui/src/main/java/org/datacleaner/windows/DataCloudLogInWindow.java]: I think I like more the previous sentence.

**Comment by ClaudiaPHI** [comment]
Works as intended. I have a single comment about a the some text for the user

File: desktop/ui/src/main/java/org/datacleaner/windows/DataCloudLogInWindow.java (hunk 1 above)
Comment (khouzvicka) [inline desktop/ui/src/main/java/org/datacleaner/windows/DataCloudLogInWindow.java]: Ok. Thanks.

**Comment by ClaudiaPHI** [comment]
LGTM!

**Pull request merged**